lib/src/components/NxPagination/NxPagination.tsx

```diff
@@ -78,33 +78,50 @@ export default function NxPagination({ className, pageCount, currentPage, onChan
           const selected = num === currentPage,
               classes = getBtnClasses(selected);
 
+          let ariaLabel;
+          if (num === 0) {
+            ariaLabel = 'goto first page';
+          }
+          else if (num === pageCount - 1) {
+            ariaLabel = 'goto last page';
+          }
+          else {
+            ariaLabel = `goto page ${num + 1}`;
+          }
+
           return (
-            <NxButton onClick={selected ? undefined : handleBtnClick(num)}
-                      tabIndex={selected ? -1 : 0 /* selected page is not focusable */}
+            <NxButton onClick={selected ? () => {} : handleBtnClick(num)}
+                      tabIndex={0}
```

Comment:
Isn't this the default for buttons? Is this actually necessary?

Reply:
2bd2e8af0fa88ea99a9e63845b2374bcf22e17cb